II.pass-tests/ES-6-practice/es6.js

```diff
@@ -1,33 +1,33 @@
-// Turn this function into an arrow function
-function arrow() {
-  return "es6 is awesome!";
-}
+// // Turn this function into an arrow function
+// function arrow() {
+//   return "es6 is awesome!";
+// }
 
-//Use the es6 syntax for default parameter
-function defaultParameter(name) {
-  var name = name || "sam";
-  return name;
-}
+// //Use the es6 syntax for default parameter
+// function defaultParameter(name) {
+//   var name = name || "sam";
+//   return name;
+// }
 
-// Use the spread operator to combine arr1 and arr2
-function combineArrays(arr1, arr2) {
-  return arr1.concat(arr2);
-}
+// // Use the spread operator to combine arr1 and arr2
+// function combineArrays(arr1, arr2) {
+//   return arr1.concat(arr2);
+// }
 
-//use destructuring to return the object's cyf property
-function destructuring(obj) {
-  return obj.cyf;
-}
+// //use destructuring to return the object's cyf property
+// function destructuring(obj) {
+//   return obj.cyf;
+// }
 
-// use template literal to return a string with the sum of a and b
-function templateString(a, b) {
-  return "The sum is equal to " + (a + b).toString();
-}
+// // use template literal to return a string with the sum of a and b
+// function templateString(a, b) {
+//   return "The sum is equal to " + (a + b).toString();
+// }
 
 module.exports = {
-  arrow,
-  defaultParameter,
-  combineArrays,
-  destructuring,
-  templateString
+  arrow: () => "es6 is awesome!",
+  defaultParameter: (name = 'sam') = name,
+  combineArrays: (arr1, arr2) => arr1.concat(arr2),
+  destructuring : (obj = {}) => obj.cyf,
+  templateString: (a, b) => "The sum is equal to " + (a + b).toString()
```

Comment:
This also isn't quite right. The exercise asked you to use a template string (with the ` character)

Reply:
Changed!